src/orion/core/io/database/mongodb.py

```diff
@@ -330,3 +330,13 @@ def _sanitize_attrs(self):
             self.options["authSource"] = settings["options"].get(
                 "authsource", self.name
             )
+
+    @classmethod
```

Comment:
We could rename this `defaults` instead and use it inside `__init__` to make sure the defaults are coherent. Otherwise we are starting to duplicate the defaults and they may diverge. (defaults here vs defaults in orion.config.storage.database)

Reply:
I didn't understand here, what should be renamed to `defaults` ?

Reply:
Oh sorry, the name of the class method, `get_defaults`.

Reply:
Ok ! Renamed `get_arguments()` to `get_defaults()` (in derived classes, too).

How should I use it inside `__init__` ?

Reply:
Inside the init, if a value is set to None or is an empty string, replace with value in `get_defaults`. Also you will need to use the default values as currently set in config here: https://github.com/Epistimio/orion/blob/develop/src/orion/core/__init__.py#L91. This way we harmonize the defaults. 
See @abergeron 's PR for the defaults on PickledDB: https://github.com/Epistimio/orion/pull/585/files#diff-f4fa1b61f95de078bc5673f3d914867b67cef706344b651be8ee5cbeb2d17ad7R102

Reply:
Ok ! Done !